.github/workflows/build.yml

```diff
@@ -11,14 +11,15 @@ on:
 
 jobs:
   build-test:
-    runs-on: ubuntu-latest
+    # Python version < 3.7 requires ubuntu-20.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [2.7, 3.5]
+        python-version: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11", "3.x"]
```

Comment:
Does it make sense to support Python versions that are officially [end-of-life](https://www.python.org/doc/sunset-python-2/) ?

Reply:
I personally would drop them, but I can understand why at least 2.7 is tested.

Reply:
OK, no problem.